style.css

```diff
@@ -0,0 +1,1106 @@
+  .single-latest:not(:last-child) {
+    margin-right: 0;
+  }
+
+  .single-latest:not(:first-child) {
+    margin-left: 2.7rem;
+  }
+
+  .hidding {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 1200px) {
+  .navbar-nav > * {
+    padding: 0 2rem !important;
+    cursor: pointer;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    transition: all 0.3s;
+  }
+
+  .navbar-nav__nav-item {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  .navbar-nav__link:link,
+  .navbar-nav__link:visited {
+    color: rgba(0, 0, 0, 0.4);
+    text-decoration: none;
+  }
+
+  .navbar-nav__toggle {
+    display: none;
+  }
+}
+
+@media only screen and (max-width: 1200px) {
+  .navbar-nav__nav-item {
+    display: none;
+  }
+
+  .header {
+    display: none;
+  }
+
+  .navbar-right {
+    margin-right: 7rem;
+  }
+
+  .search-btn {
+    border-right: 2px solid rgba(0, 0, 0, 0.1);
+  }
+
+  .news-title {
+    justify-content: center;
+  }
+
+  .news-title h2 {
+    align-self: center;
+    margin-left: 0;
+  }
+
+  .latest-title {
+    flex-direction: column;
+    margin-bottom: 1rem;
+    align-items: center;
+  }
+
+  .lates-top,
+  .latest-bottom {
+    flex-direction: column;
+    flex-wrap: nowrap;
+    align-items: center;
+  }
+
+  .single-latest:not(:first-child) {
+    margin-left: 0;
+  }
+}
+
+@media only screen and (max-width: 900px) {
+  html {
+    font-size: 50%;
+  }
+
+  .grid-header {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: 50rem 25rem;
+    margin: 0.7rem;
+    gap: 0.7rem;
+  }
+
+  .grid-header-1 {
+    grid-column: 1 / 3;
+    grid-row: 1 / 2;
+  }
+
+  .grid-header-2 {
+    grid-column: 1 / 2;
+    grid-row: 2 / 3;
+  }
+
+  .grid-header-3 {
+    grid-column: 2 / 3;
+    grid-row: 2 / 3;
+  }
+}
+
+@media only screen and (max-width: 828px) {
+  .rounds-links {
+    flex-direction: column;
+  }
+
+  .rounds-links a {
+    padding-bottom: 1rem;
+  }
+
+  .rounds-links a:not(:last-child) {
+    border-right: 1px solid transparent;
+  }
+
+  .single-round {
+    width: 100%;
+  }
+
+  .single-popular {
+    width: 90%;
+  }
+}
+
+@media only screen and (max-width: 500px) {
+  .grid-header {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50rem repeat(2, 25rem);
+    margin: 0.7rem;
+    gap: 0.7rem;
+  }
+
+  .grid-header-1 {
+    grid-column: 1 / 2;
+    grid-row: 1 / 2;
+  }
+
+  .grid-header-2 {
+    grid-column: 1 / 2;
+    grid-row: 2 / 3;
+  }
+
+  .grid-header-3 {
+    grid-column: 1 / 2;
+    grid-row: 3 / 4;
+  }
+
+  .single-news {
+    width: 100%;
+  }
+
+  .single-latest {
+    width: 60%;
+    display: block;
+    margin: 0 auto;
+  }
+
+  .news-img-box,
+  .latest-img-box {
+    width: 8rem;
+    height: 8rem;
+    float: left;
+    margin-right: 1rem;
+  }
+
+  .single-news h3,
+  .single-latest h3 {
+    font-size: 1.6rem;
+  }
+
+  .single-news p,
+  .single-latest p {
+    font-size: 1.4rem;
+  }
+}
```

Comment:
Instead of using all these media queries, you can use only 3 media queries by adding the following syntax:

`@media only screen and (min-width: ???px) and (max-width: ???px) { ... }`

Check out the pathwright specifications for details:

![image](https://user-images.githubusercontent.com/2739245/88444411-91c2ff00-cde2-11ea-8744-33df024bad63.png)
![image](https://user-images.githubusercontent.com/2739245/88444414-9687b300-cde2-11ea-8189-6d09edcf8040.png)